the-badass-todo-list-part-1/input.js

```diff
@@ -62,6 +67,7 @@ var TodoInputManager = {
   }
 };
 
+// Quiz#3
```

Comment:
You know there are thousands code like this torture people everyday. So it's a "real" case, and that's what I want you to catch